spec/ace.md

```diff
@@ -264,9 +266,9 @@ Retrievable at `$AC_METADATA_URL/acMetadata/v1/pod`
 
 | Entry       | Description |
 |-------------|-------------|
-|annotations/ | Top level annotations from Pod Manifest. |
+|annotations/ | Top level annotations from Pod Manifest. Response body should conform to the sub-schema of the annotations property from the Pod specification (e.g. ```[ { "name": "ip-address", "value": "10.1.2.3" } ]```). |
```

Comment:
That's not what happens now (at least as implemented by `rkt`). It returns `text/plain` with just the annotation names, separated by newlines. That is inline with AWS metadata service -- the inspiration for this metadata service. However if want to move all responses to `application/json`, I would be OK with it.